wikitemplate.md

```diff
@@ -22,21 +22,21 @@
 
 ### Extensions/Plugins
 
-- [ ] Verify pdfium, Torrent viewer extensions are installed automatically on fresh profile and cannot be disabled
+- [ ] Verify pdfium, Torrent viewer extensions are installed automatically on fresh profile and cannot be disabled (they don't show up in `brave://extensions`)
 - [ ] Verify older version of an extension gets updated to new version via Google server
-- [ ] Verify older version of an extension gets updated to new version via Brave server 
-- [ ] Verify that `magnet` links and `.torrent` files correctly opens WebTorrent and you're able to download the file(s)
-  - **Tip:** Free torrents available via https://webtorrent.io/torrents/
+- [ ] Verify older version of an extension gets updated to new version via Brave server
+- [ ] Verify that `magnet` links and `.torrent` files correctly open WebTorrent and you're able to download the file(s)
+  - **Tip:** Free torrents available via https://torrent.ubuntu.com/
 
-### CWS
+### Chrome Web Store (CWS)
 
 - [ ] Verify that installing https://chrome.google.com/webstore/detail/adblock-plus-free-ad-bloc/cfhdojbkjhnklbpkdaibdccddilifddb from CWS displays the `Brave has not reviewed the extension.` warning via the "Add Extension" modal
 - [ ] Verify that installing https://chrome.google.com/webstore/detail/lastpass-free-password-ma/hdokiejnpimakedhajhdlcegeplioahd from CWS doesn't display the `Brave has not reviewed the extension.` warning via the "Add Extension" modal
 
 ### Flash
 
 - [ ] Test that Flash gets blocked by default when installed
-- [ ] Test that once you allow Flash, it turns into a click to play area, and loads flash when allowed
+- [ ] Test that once you allow Flash, it turns into a click-to-play area, and loads Flash when allowed
 
```

Comment:
Better to remove the section itself as its obsolete now.

Reply:
Thanks, @srirambv!  Happily did so in https://github.com/brave/qa-resources/pull/203/commits/6b24215fb9bfc7d984ccfd114021f1151ad64ca7.